recipe/meta.yaml

```diff
@@ -14,13 +14,15 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
   noarch: python
 
 requirements:
   build:
```

Comment:
I think you can get rid of the build section and add python to the host section.

Reply:
The automatic #7 seems to have worked, so we could merge that and go from there